example-app/src/main/res/values/strings.xml

```diff
@@ -39,6 +41,9 @@
     <string name="split_card_funding_sources_title">Split Card Funding Sources</string>
     <string name="enable_card_address_form_key">enable_card_address_form</string>
     <string name="enable_card_address_form_title">Enable card address form</string>
+    <string name="instant_payment_method_type_key">instant_payment_method_type</string>
+    <string name="instant_payment_method_type_title">Instant Payment Method Title</string>
```

Comment:
The string should be `Instant Payment Method Type` instead of `Instant Payment Method Title` right?

Reply:
True!